Units/simple-forth.d/expected.tags

```diff
@@ -0,0 +1,9 @@
+DUMMY1	input.fth	/^10 constant DUMMY1$/;"	c
```

Comment:
I have a plan to add another test case.
So could you rename `Units/simple-forth.d` to `Units/parser-forth.r/simple-forth.d`?
So I can add a case like `Units/parser-forth.r/masatake.d`.